Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9.0-alpine3.12
+
+RUN pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib
+
+COPY ./requirements.txt ./requirements.txt
+
+RUN pip install -r requirements.txt
+
+COPY ./config.json ./config.json
+COPY ./export.json ./export.json
```

Comment:
This file is not well documented, but from what I understand, it's a runtime file, so shouldn't be in the image.

Reply:
done